### DISCUSSION TITLE
Make lists discoverable

### DISCUSSION THREAD

**Discussion by llagerlof** [community/community#8661] [Other Feature Feedback, Questions, & Ideas] [answered]
Some users, like me, love to discover new projects. Would be fantastic to browse a public user created lists repository to find new exciting programs. Currently to find lists content we need to access a specif user profile.

**Comment by brianlovin** [comment, marked as answer]
Thanks for the suggestion @llagerlof — totally agree, and this is something we're hoping to work on soon. We want to make it easier to discover other great lists all across GitHub.

**Reply by Ahmadtaqwa58** [reply]
109

**Reply by Mazzika1** [reply]
Okay

**Reply by Raunaksplanet** [reply]
Sounds Nice 😁

**Reply by Shai69bb** [reply]
👍

**Reply by Boagey** [reply]
1

**Reply by 00676** [reply]
Nice

**Reply by Katyjones444** [reply]
I'm new here probably something I can't do or need can someone explain what this gitfit is ?

**Reply by SiThuKyawTint-GuGu** [reply]
Nice

**Reply by Rezakhani48** [reply]
Thank you deer
Problam me error 1015 
What to do?

**Reply by Skippz88** [reply]
Denial-of-Service (DoS) attacks: Attackers may flood a server with a massive number of requests, exhausting its resources and rendering it unavailable to legitimate users. Rate limiting can help mitigate the impact of such attacks by limiting the number of requests an attacker can make within a given time frame.

Throttling abusive clients: Sometimes, certain clients or applications may send an excessive number of requests, intentionally or unintentionally, causing a strain on the server. Rate limiting allows the server to restrict the number of requests from such clients, preventing them from monopolizing server resources.

Protecting against brute-force attacks: Rate limiting can be employed to prevent brute-force attacks, where an attacker attempts to guess passwords or access restricted areas of a website by trying numerous combinations. By limiting the number of login attempts within a specific time period, rate limiting can make these attacks significantly more difficult.

**Comment by TajamalHussain03226495507** [comment]
Aviator single

**Reply by TajamalHussain03226495507** [reply]
Reply

**Comment by PDKIDuski0878** [comment]
[۲۳/۰۴/۲۴ ۲۳:۳۴:۰۴] Running on vivo V2035(2036); Supported ABIs: [arm64-v8a, armeabi-v7a, armeabi]; Android: 13; TIRAMISU; SDK 33; Device Rooted: NO
[۲۸/۰۴/۲۴ ۲۳:۰۰:۱۴] Starting V2Ray Core
[۲۸/۰۴/۲۴ ۲۳:۰۰:۱۴] Version: Xray Core 1.6.1
[۲۸/۰۴/۲۴ ۲۳:۰۰:۱۴] <b><font color='#13DF04'>WakeLock acquired</font></b>
[۲۸/۰۴/۲۴ ۲۳:۰۰:۱۴] Binding HTTP address to 10.22.10.1:8080
[۲۸/۰۴/۲۴ ۲۳:۰۰:۱۴] Binding SOCKS address to 10.22.10.1:1080
[۲۸/۰۴/۲۴ ۲۳:۰۰:۱۴] Using Default DNS
[۲۸/۰۴/۲۴ ۲۳:۰۰:۱۴] Preparing VPN routes
[۲۸/۰۴/۲۴ ۲۳:۰۰:۱۴] [0.0.0.0/5, 8.0.0.0/7, 11.0.0.0/8, 12.0.0.0/6, 16.0.0.0/4, 32.0.0.0/3, 64.0.0.0/2, 128.0.0.0/3, 160.0.0.0/5, 168.0.0.0/6, 172.0.0.0/12, 172.32.0.0/11, 172.64.0.0/10, 172.128.0.0/9, 173.0.0.0/8, 174.0.0.0/7, 176.0.0.0/4, 192.0.0.0/9, 192.128.0.0/11, 192.160.0.0/13, 192.169.0.0/16, 192.170.0.0/15, 192.172.0.0/14, 192.176.0.0/12, 192.192.0.0/10, 193.0.0.0/8, 194.0.0.0/7, 196.0.0.0/6, 200.0.0.0/5, 208.0.0.0/4, 224.0.0.0/3]
[۲۸/۰۴/۲۴ ۲۳:۰۰:۱۵] Apps filter is enabled, allowed apps: none
[۲۸/۰۴/۲۴ ۲۳:۰۰:۱۶] <b>Connected</b>
[۲۸/۰۴/۲۴ ۲۳:۰۰:۱۶] <b>VPN Service Started</b>

**Reply by Kaplan8080** [reply]
> [۲۳/۰۴/۲۴ ۲۳:۳۴:۰۴] Running on vivo V2035(2036); Supported ABIs: [arm64-v8a, armeabi-v7a, armeabi]; Android: 13; TIRAMISU; SDK 33; Device Rooted: NO
> [۲۸/۰۴/۲۴ ۲۳:۰۰:۱۴] Starting V2Ray Core
> [۲۸/۰۴/۲۴ ۲۳:۰۰:۱۴] Version: Xray Core 1.6.1
> [۲۸/۰۴/۲۴ ۲۳:۰۰:۱۴] <b><font color='#13DF04'>WakeLock acquired</font></b>
> [۲۸/۰۴/۲۴ ۲۳:۰۰:۱۴] Binding HTTP address to 10.22.10.1:8080
> [۲۸/۰۴/۲۴ ۲۳:۰۰:۱۴] Binding SOCKS address to 10.22.10.1:1080
> [۲۸/۰۴/۲۴ ۲۳:۰۰:۱۴] Using Default DNS
> [۲۸/۰۴/۲۴ ۲۳:۰۰:۱۴] Preparing VPN routes
> [۲۸/۰۴/۲۴ ۲۳:۰۰:۱۴] [0.0.0.0/5, 8.0.0.0/7, 11.0.0.0/8, 12.0.0.0/6, 16.0.0.0/4, 32.0.0.0/3, 64.0.0.0/2, 128.0.0.0/3, 160.0.0.0/5, 168.0.0.0/6, 172.0.0.0/12, 172.32.0.0/11, 172.64.0.0/10, 172.128.0.0/9, 173.0.0.0/8, 174.0.0.0/7, 176.0.0.0/4, 192.0.0.0/9, 192.128.0.0/11, 192.160.0.0/13, 192.169.0.0/16, 192.170.0.0/15, 192.172.0.0/14, 192.176.0.0/12, 192.192.0.0/10, 193.0.0.0/8, 194.0.0.0/7, 196.0.0.0/6, 200.0.0.0/5, 208.0.0.0/4, 224.0.0.0/3]
> [۲۸/۰۴/۲۴ ۲۳:۰۰:۱۵] Apps filter is enabled, allowed apps: none
> [۲۸/۰۴/۲۴ ۲۳:۰۰:۱۶] <b>Connected</b>
> [۲۸/۰۴/۲۴ ۲۳:۰۰:۱۶] <b>VPN Service Started</b>
>

**Comment by smilefox-lab** [comment]
I think one way to facilitate this is through the use of GitHub Awesome Lists or GitHub Topic tags.

**Reply by vigvuv-jozNa4-hycnas** [reply]
ウk[dl]()

**Comment by vigvuv-jozNa4-hycnas** [comment]
##　　　　　　
F we go owe him a

**Comment by llagerlof** [comment]
"soon" has a totally different meaning for developers 😂

**Reply by vigvuv-jozNa4-hycnas** [reply]
- - ######- - [ ] [hel@[_@_]()]()[_**@**```
> ````
```****_]()

**Reply by PDKIDuski0878** [reply]
داداش این موال v2ray
چرا کار نمیکنهپ

**Reply by PDKIDuski0878** [reply]
> - - ######- - [ ] [hel@[_@_]()]()[_**@**```
> > ````
> ```****_]()

داداش کار نمیکنه مگه v2ray نیست

**Comment by Wladimir5** [comment]
> Aviator single

**Comment by RemadanAdusebro** [comment]
It's very wonderful

**Comment by JeanLeeJeffrson** [comment]
It's about creating just an idea for a software or application and allowing users to create it with building and implementation with a github recommendation to create a maintainer, or pass verification to github? 🦥😇

**Comment by limcheewee** [comment]
> Some users, like me, love to discover new projects. Would be fantastic to browse a public user created lists repository to find new exciting programs. Currently to find lists content we need to access a specif user profile.

**Comment by 00676** [comment]
Nice☮️

**Comment by Udlyc008** [comment]
****

**Comment by schnuff2702** [comment]
****

**Comment by Mazzika1** [comment]
Okay

في الثلاثاء، ٢٨ مايو ٢٠٢٤, ٨:٥٨ م schnuff2702 ***@***.***>
كتب:

**Comment by DastagirHossion** [comment]
Alaway thinking any work

**Comment by Boagey** [comment]
> Some users, like me, love to discover new projects. Would be fantastic to browse a public user created lists repository to find new exciting programs. Currently to find lists content we need to access a specif user profile.

**Comment by 00676** [comment]
Let's go

**Comment by Nurtrisariyanto** [comment]
Uhuiyyyÿ

**Comment by Jacques8787** [comment]
Agreed. Excellent idea. Would love to see a public user created lists repository.

**Comment by Mustangclayton** [comment]
Build the discovery project waves that branches grow from

**Comment by Skippz88** [comment]
It's about creating just an idea for a software or application and allowing users to create it with building and implementation with a github recommendation to create a maintainer, or pass verification to github? 🦥😇
